### ISSUE TITLE
Code for maps

### ISSUE THREAD

**Issue by dirkschumacher** [R4EPI/surveys#5]
It seems maps are useful in all surveys, however not always used. I suppose some general code snippets for map generation might be useful and should be part of the report templates.

Any thoughts?

**Comment by dirkschumacher** [comment]
So far I saw one map (a simple geographic ares with points).

**Comment by epiamsterdam** [comment]
Yes. Agree. The main issue is related to the way that geographic data is collected as part of outbreak linelists. We might not actually know where the villages of residence are that people report, so unlikely we have an exact location to pinpoint them to. In some cases we might be able to narrow it down to a higher administrative level (i.e. ward or health zone), in which case we could do some incidence rates mapping.
The spot map you saw from chad for HEV is because we also collected GPS points for each of the cases we identified. That is rare, but is increasing in frequency that we have that ability - particularly with mobile data collection.

**Comment by dirkschumacher** [comment]
Note to self: For a first version we just provide code to get a shape file and draw some points.
